File: src/main/java/org/jukeboxmc/item/behavior/ItemPumpkinPie.java
package org.jukeboxmc.item.behavior;

import org.jukeboxmc.item.ItemType;
import org.jukeboxmc.util.Identifier;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class ItemPumpkinPie extends ItemFood {

    public ItemPumpkinPie( Identifier identifier ) {
        super( identifier );
    }

    public ItemPumpkinPie( ItemType itemType ) {
        super( itemType );
    }

    @Override
    public float getSaturation() {
        return 4.8f;
    }

    @Override
    public int getHunger() {
        return 8;
    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/item/behavior/ItemPumpkinPie.kt
package org.jukeboxmc.server.item.behavior

import org.jukeboxmc.api.item.ItemType

class ItemPumpkinPie(itemType: ItemType, countNetworkId: Boolean) : JukeboxItemFood(itemType, countNetworkId) {

    override fun getSaturation(): Float {
        return 4.8F
    }

    override fun getHunger(): Int {
        return 8
    }
}